index.html

```diff
@@ -79,97 +85,103 @@ <h6 class="h6-documentation">Skills</h6>
   </section>
   <!--Documentation section-->
   <hr>
-  <div class="j-content">
-    <img class="icons" src="img/github-icon.svg" alt="Github logo">
-    <img class="icons" src="img/linkedin-icon.svg" alt="Github logo">
-    <img class="icons" src="img/two-fingers-up.svg" alt="hand oing peace and love">
-    <img class="icons" src="img/twitter-icon.svg" alt="twitter logo">
-    <img class="icons" src="img/medium-icon.svg" alt="medium logo">
+  <div class="c-container w-class">
+    <div class="j-content p-work hidden-el">
+      <img src="img/arrow-down.svg" alt="arrow down" id="arrow-down">
+      <a class="w-style pff-500">WORK</a>
+    </div>
+    <div class="j-content">
+      <img class="icons" src="img/github-icon.svg" alt="Github logo">
+      <img class="icons" src="img/linkedin-icon.svg" alt="Github logo">
+      <img class="icons" src="img/two-fingers-up.svg" alt="hand oing peace and love">
+      <img class="icons" src="img/twitter-icon.svg" alt="twitter logo">
+      <img class="icons" src="img/medium-icon.svg" alt="medium logo">
+    </div>
+    <div class="j-content p-work ">
+      <img src="img/arrow-down.svg" alt="arrow down" id="arrow-down1">
+      <a class="w-style pff-500">WORK</a>
+    </div>
   </div>
   <!--My Work section-->
   <section id="myWork">
```

Comment:
- [ ] Kindly make sure that the elements inside this section are centered. Right now, they are closer to the left side of the screen for the desktop version, which makes the page look irregular, so please find the bug and fix this issue.
- [ ] Also, if you check the template that you are given, you will see that for each row, the position of the image and text changes. For the first 'work-card' you have the image to the left and text to the right, and they swap for the second row and they keep swapping, but in your design, they are all centered with the text at the bottom. Please make sure to fix this issue in order to stick to the design as much as you can.

Reply:
Figma template
![image](https://user-images.githubusercontent.com/98621809/154823095-568aa8b9-dccb-4869-85be-257816811329.png)

Mobile version from inspector
![image](https://user-images.githubusercontent.com/98621809/154823109-6fb5fdbc-74e6-481e-9e04-b8e1a34c712d.png)

Desktop version from inspector
![image](https://user-images.githubusercontent.com/98621809/154823124-51868ad3-8302-4fe4-be69-73c2a5a6c9ef.png)